src/jit/core_templates.expr

```diff
@@ -137,13 +182,79 @@
       (carg (tc) ptr)
       (carg $1 ptr)) ptr_sz))
 
+(template: exception
+  (let: (($ah (^getf (tc) MVMThreadContext active_handlers)))
+    (if
+      (nz $ah)
+      (^getf $ah MVMActiveHandler ex_obj)
+      (^vmnull))))
+
+(template: bindexpayload!
+  (callv (^func &MVM_bind_exception_payload)
```

Comment:
In this case, the  'destructive' '!' is redundant, since this is a read-only operand anyway. (It doesn't change the $0, which is why you get that as a value and not as an address). It isn't actually harmful (it has no effect at all)